### PR TITLE
[fix](group_commit) Fix check auth error when relaying wal

### PR DESCRIPTION
**Pull request by hust-hhb**
Now relaying wal will be failed because checking auth code has been changed on fe, this pr fix this problem.
Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hust-hhb** [comment]
run buildall

File: be/src/olap/wal/wal_table.cpp [hunk 1]
```diff
@@ -278,8 +278,10 @@ Status WalTable::_handle_stream_load(int64_t wal_id, const std::string& wal,
     std::shared_ptr<StreamLoadContext> ctx = std::make_shared<StreamLoadContext>(_exec_env);
     ctx->sql_str = sql_str;
     ctx->wal_id = wal_id;
-    ctx->auth.auth_code = wal_id;
     ctx->label = label;
+    ctx->token = "relay_wal"; // this is a fake, fe not check it now
```
Comment (mymeiyi) [inline be/src/olap/wal/wal_table.cpp:282]: use ctx->auth.token ?

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
already change the code,take a look please @mymeiyi

**Review by mymeiyi — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

**Pull request merged**